tools/local-network-setup/README.md

```diff
@@ -1,78 +1,76 @@
-DKG local network setup tool
-========================
+# DKG local network setup tool
 
-This tool will help you set up a local DKG v6 network running with the Ganache blockchain. It is useful for development and testing purposes and is used internally by the OriginTrail core developers.
+This tool will help you set up a local DKG v6 network running with the Hardhat blockchain. It is useful for development and testing purposes and is used internally by the OriginTrail core developers.
 <br/>
 
-
 **Note: This tool is an internal tool used by the OriginTrail team and thus is developed for our workflow, meaning that it currently only supports MacOS**, but we encourage you to adapt it for your workflow as well.
 
+# Prerequisites
 
-Prerequisites
-=============
-
-* An installed and running triplestore (graph database)
-  * We recommend testing with GraphDB. In order to download GraphDB, please visit their official [website](https://graphdb.ontotext.com/). Alternatively other triple stores can be used (Blazegraph, Apache Jena and other RDF native graph databases)
-* An installed and running MySQL server
-* You should have installed npm and Node.js (v16) or higher
-
+-   An installed and running triplestore (graph database)
+    -   We recommend testing with Blazegraph. In order to download Blazegraph, please visit their official [website](https://blazegraph.com/). Alternatively other triple stores can be used (GraphBD, Apache Jena and other RDF native graph databases)
+-   An installed and running MySQL server
+-   You should have installed npm and Node.js (v16) or higher
 
 # Setup instructions
 
 In order to run the local network you fist need to clone the "ot-node" repository.
 <br/>
 
 ## 1. CLONE OT-NODE REPOSITORY & INSTALL DEPENDENCIES
+
 After cloning the **ot-node** repository, please checkout to "v6/develop" branch and install dependencies by running:
+
 ```bash
 git clone https://github.com/OriginTrail/ot-node.git && cd ot-node/ && npm install && cd ..
 ```
+
 <br/>
 
 ### 2.2 Create the .env file inside the "ot-node" directory:
+
 ```bash
 nano .env
 ```
+
 and paste the following content inside (save and close):
+
 ```bash
 NODE_ENV=development
 RPC_ENDPOINT_BC1=http://localhost:8545
 RPC_ENDPOINT_BC2=http://localhost:9545
 PRIVATE_KEY=02b39cac1532bef9dba3e36ec32d3de1e9a88f1dda597d3ac6e2130aed9adc4e
```

Comment:
I don't think it's used anywhere

Reply:
Yeah we don't use it anywhere we can remove it